code/datums/action.dm

```diff
@@ -587,7 +589,14 @@
 	if(progress == 1)
 		return ..() // This means that the spell is charged but unavailable due to something else
 
-	var/alpha = 220 - 140 * progress
+	var/alpha = 220
+	alpha = 220 - 140 * progress
```

Comment:
Why give the var/alpha a value if you are going to override it in the next line? Its initial value is being requested in another place?

Reply:
no, thats a mistake